Wrappers/Python/ccpi/optimisation/operators/FiniteDifferenceOperator.py

```diff
@@ -74,7 +402,7 @@ def __init__(self, gm_domain, gm_range=None, direction=0, bnd_cond = 'Neumann'):
         #self.voxel_size = kwargs.get('voxel_size',1)
         # this wrongly assumes a homogeneous voxel size
 #        self.voxel_size = self.gm_domain.voxel_size_x
-        super(FiniteDiff, self).__init__(domain_geometry=gm_domain, 
+        super(OldFiniteDiff, self).__init__(domain_geometry=gm_domain, 
```

Comment:
I would remove the old one completely. It just complicates the code keeping it around.

Reply:
done